mkdocs.yml

```
@@ -479,6 +479,7 @@ nav:
                     -   "Diagram Root README": 02-architecture/diagrams/README.md
                     -   "Historical Diagram Catalog": 02-architecture/diagrams/governance/diagram-catalog.md
                     -   "Class Diagram Description Overview": 02-architecture/diagrams/descriptions/class-summary.md
+                    -   "ChEMBL Activity Dataflow Passport": 02-architecture/generated/pipeline-dataflows/chembl_activity/pipeline-passport.md
```

YAMLlint (1.37.1)

[error] too many spaces after hyphen

(hyphens)